configure.ac

```diff
@@ -438,6 +430,16 @@ AM_CONDITIONAL([FPM], [test "x$enable_fpm" = "xyes"])
 
 if test "x${enable_dev_build}" = "xyes"; then
    AC_DEFINE(DEV_BUILD,,Build for development)
+   AC_C_FLAG([-g])
```

Comment:
Wanna make that -g3? we hava a lot of macros, might as well have it on in a dev build

Reply:
of course that might not be supported by all compilers/debuggers, but we are already using GNU extensions anyway.